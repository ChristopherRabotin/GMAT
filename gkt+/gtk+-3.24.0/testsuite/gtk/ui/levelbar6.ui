<!-- invalid element in levelbar markup -->
<interface>
  <object class="GtkLevelBar">
    <offsets>
    </offsets>
    <warbl/>
  </object>
</interface>
