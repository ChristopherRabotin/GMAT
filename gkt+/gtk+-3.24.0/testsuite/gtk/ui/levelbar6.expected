ERROR: gtk-builder-error-quark 1
<input>:6:13 Unhandled tag: <warbl>
